src/lib/viewers/BaseViewer.js

```diff
@@ -284,6 +297,31 @@ class BaseViewer extends EventEmitter {
         this.destroyed = true;
     }
 
+    /**
+     * Handles a download error when using a non default host.
+     *
+     * @param {Error} err - Load error
+     * @param {string} downloadURL - download URL
+     * @return {void}
+     */
+    handleDownloadError(err, downloadURL) {
+        if (this.haveRetried) {
+            /* eslint-disable no-console */
+            console.error(err);
```

Comment:
If you want to put `console.error(err)` inside of the code that emits `preview_error` events, we can throttle back the number of `console` statements that exist in our codebase! Up to you :)

Reply:
I do! But we would need to remove them all in one to avoid double printing, so I'm leaving this for now and then we can go back and do a commit where we grab them all.